packages/@aws-cdk-testing/framework-integ/test/aws-synthetics/test/integ.canary-auto-delete-lambda.ts

```diff
@@ -10,7 +10,7 @@ class TestStack extends Stack {
 
     new synthetics.Canary(this, 'Canary', {
       canaryName: 'next',
-      runtime: synthetics.Runtime.SYNTHETICS_NODEJS_PUPPETEER_4_0,
+      runtime: synthetics.Runtime.SYNTHETICS_NODEJS_PUPPETEER_7_0,
```

Comment:
`Runtime.SYNTHETICS_NODEJS_PUPPETEER_4_0` was causing the integ test to fail due to this being a deprecated runtime. The error message was as follows:

```
Invalid request provided: Deprecated runtime version specified.
```

I've updated the runtime to the latest.